examples/tv-casting-app/android/App/app/src/main/jni/cpp/TvCastingApp-JNI.cpp

```diff
@@ -153,7 +155,8 @@ JNI_METHOD(jboolean, initServer)(JNIEnv * env, jobject, jobject jCommissioningCo
     CHIP_ERROR err = SetUpMatterCallbackHandler(env, jCommissioningCompleteHandler, gCommissioningCompleteHandler);
     if (err == CHIP_NO_ERROR)
     {
-        CastingServer::GetInstance()->InitServer(CommissioningCompleteHandler);
+        // CHIP Server was already initialized for this Android app
```

Comment:
what makes this comment true? I don't see it following from arguments. Please explain like:

`Commissioning window should already be open by the time this method is called, so not open it again)`

(not sure this is the case ... please make sure your comment explains why this is done)

Reply:
As discussed above, code has been refactored, so we don't need any special flag passing now.